python/CMakeLists.txt

```diff
@@ -1,14 +1,19 @@
-find_package(pybind11 2.2 CONFIG)
+find_package(pybind11 2.11 CONFIG)
```

Comment:
Increased the version from 2.2 to 2.11 here

Reply:
same comment as in the opm-common PR.